snapshot/controllers.go

```diff
@@ -0,0 +1,92 @@
+package snapshot
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/labstack/echo/v4"
+)
+
+func Topology(c echo.Context) error {
+	id := c.Param("id")
+	related, _ := strconv.ParseBool(c.QueryParam("related"))
+	logStart := c.QueryParam("logStart")
+	logEnd := c.QueryParam("logEnd")
```

Comment:
Can we rename these to just start and end to be consistent, as well as in the function args ?